Zend/zend.h

```diff
@@ -340,6 +341,7 @@ extern void (*zend_printf_to_smart_string)(smart_string *buf, const char *format
 extern void (*zend_printf_to_smart_str)(smart_str *buf, const char *format, va_list ap);
 extern ZEND_API char *(*zend_getenv)(const char *name, size_t name_len);
 extern ZEND_API zend_string *(*zend_resolve_path)(zend_string *filename);
+extern ZEND_API zend_result (*zend_os_csprng_random_bytes)(void *bytes, size_t size, char *errstr, size_t errstr_size);
```

Comment:
I would made/name this ``void zend_random_bytes(void *bytes, size_t size)`` and moved the fall-back splitmix64 implementation into ext/random as well, but I'm not sure. If you think, the current name/prototype is better - keep it.

Reply:
I've kept this name and added `zend_general_random_bytes()`. It implements the general purpose PRNG and seeding with fallback. I was able to remove all the rand code from zend_alloc.